tests/test_runpath.py

```diff
@@ -0,0 +1,31 @@
+import nose
+import os
+import shutil
+import tempfile
+
+import cle
+
+TEST_BASE = os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                                         os.path.join('..', '..', 'binaries'))
+
+def test_runpath():
+    tempdir = tempfile.mkdtemp()
+    try:
+        runpath_file = os.path.join(TEST_BASE, 'tests', 'x86_64', 'runpath')
+        relocated_file = os.path.join(tempdir, 'runpath')
+
+        expected_libs = []
+        os.mkdir(os.path.join(tempdir, 'lib'))
+        for lib in ["libc.so.6", "ld-linux-x86-64.so.2"]:
+            src = os.path.join(TEST_BASE, 'tests', 'x86_64', lib)
+            dst = os.path.join(tempdir, 'lib', lib)
+            expected_libs.append(os.path.realpath(dst))
+            shutil.copy(src, dst)
+
+        shutil.copy(runpath_file, relocated_file)
+
+        loader = cle.Loader(relocated_file, except_missing_libs=True)
```

Comment:
Are tests only supposed to run on Linux?

Reply:
Tests are also supposed to run on windows, but I don't think that'll be a problem here? All the libraries are bundled in `binaries` and then copied to the destination, so they should all be there on windows too.